submission-instructions.md

```diff
@@ -12,13 +12,15 @@ by the submission system and/or subject to an administrative reject.
     review by another conference or journal. You may submit material
     presented previously at a workshop without copyrighted proceedings.
 
--   The submission is limited to eleven (11) pages in the IEEE 8.5" x
-    11" (US letter size paper) Two-Column Format using 10pt font, with
-    no more than 6 lines per inch. This page limit applies to all
+-   The submission is limited to eleven (11) pages in the ACM 8.5" x
+    11" (US letter size paper) using 9pt font, with
+    no more than 7 lines per inch. This page limit applies to all
     content NOT INCLUDING references, and there is no page limit for
     references. Your paper must print satisfactorily on both Letter
     paper (8.5"x11") and A4 paper (8.27"x11.69"). The box containing the
-    text should be no larger than 7.15"x9.27" (18.2cm x 23.5cm).
+    text should be no larger than 7.15"x9" (18.2cm x 22.9cm).
+    Note that the supplied Word template uses single-column layout,
+    while the LaTeX template uses two-column layout.
```

Comment:
This inconsistency again stems from the templates coming from https://www.acm.org/publications/taps/word-template-workflow. I assume they have reasonable reasons for that, but it may result in mildly inconsistent "size limits" for Word and LaTeX submissions.

Reply:
The inconsistency will create more trouble than is necessary. How about we just dump the Word template?

Reply:
Dumped: 942d2ecad0a03005d089a7f17b9e3f2559586659